src/components/nfts/NFTCardGallery.scss

```diff
@@ -0,0 +1,91 @@
+#card-gallery-container {
```

Comment:
I believe it is better to always use **_class_** instead of **_id_** for styles.
With _**id**_, It's not possible to have code reuse.